### PR TITLE
SDM image alt tags

### PR DESCRIPTION
**Pull request by ElminaIusifova**

### CODE REVIEW

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 1]
```diff
@@ -816,11 +816,13 @@ Organizing portfolios around development value streams enables visualizing the f
...
 An IT solution is a combination of one or more IT Products.
-It produces the environment within which end-users operates.
+It produces the environment within which end-users operates.  
 
-![How IT Solutions, IT Products, software, and infrastructure relate to each other]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+Figure 1. Relationship Model between the different software elements (part 1).  
 
-**IT Product**
+![Below is a flowchart with five elements connected to each other.  The flowchart has the following elements displayed in the box: IT Solution, IT Component, Software, Infrastructure. The flow goes from left to right and connects those elements with an arrow. The Configuration element is located below the Software and Infrastructure elements, the latter are pointing to it from both sides. ]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:824]: This image also appears in ABB and @gcharest  just added alt text for it as follows. Recommend consistency and using the same alt text. 
"An UML diagram representing the hierarchical relation of an IT Solution and its components.
There are 5 rectangles on the diagram connected by relationship lines.
The first rectangle to the left is titled "IT Solution".
It is connected to a second rectangle, titled "IT Product", located on its right by a relationship line; the end of the line starting from the "IT Solution" rectangle has a "1" and the other end has "1...*" and a small white diamond.
The "IT Product" rectangle is connected to a third rectangle, titled "Software", located on its right by a relationship line; the end of the line starting from the "IT Product" rectangle has a "1" and the other end has a "1...*" and a small white diamond.
The "Software" rectangle has two relationships connections.
The first connection is to a rectangle, titled "Configuration", located below the "Software" rectangle; the end of the line starting from the "Software" rectangle has a "1" and the other end has a "1...*" and a small black diamond.
The second connection is to another rectangle, titled "Infrastructure", located to the right of the "Software" rectangle; the end of the line starting from the "Software" rectangle has a "1" and the other end has a "1...*" and a small white diamond.
There is one last connection, linking the "Infrastructure" and "Configuration" rectangles; the end of the line starting from the "Infrastructure" rectangle has a "1" and the other end has a "1...*" and a small black diamond."

File: _pages/en/strategy-target-solution-delivery-model.md [hunk 2]
```diff
@@ -831,9 +833,11 @@ A “working IT Product” is a version of the IT Product which allows DevOps te
 A working IT Product must be able to be used by end-users and provide DevOps teams with observation of end-users behaviours.
 
 For the scope of this Strategy, Operating Systems are NOT defined as IT products.
-Therefore should an IT Product depend on an Operating System to run in production, it is compliance with this Guiding Policy.
+Therefore should an IT Product depend on an Operating System to run in production, it is compliance with this Guiding Policy.  
+
+Figure 2. Relationship Model between the different software elements (part 2).  
 
-![How IT Solutions, IT Products, software, and infrastructure relate to each other]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+![Below is a flowchart with five elements connected to each other.  The flowchart has the following elements displayed in the box: IT Solution, IT Component, Software, Infrastructure. The flow goes from left to right and connects those elements with an arrow. The Configuration element is located below the Software and Infrastructure elements, the latter are pointing to it from both sides.]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
```
Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:841]: Same comment as above re: alt text as this appears to be the same image?
Question for @remyb2canada ... my sleepy morning eyes are registering this as the same image as the one above. If yes, why duplicate it? Is perhaps confusing to have 'figure 1' and 'figure 2' if it is the same image.

Comment (RachelMuston) [inline _pages/en/strategy-target-solution-delivery-model.md:841]: Also I just noticed.....the bottom right box says 'configutation'.  Is that right?  Should it be 'configuration'? If yes we also need to change the image on the ABB page cc @gcharest

**Review by remyb2canada — APPROVED** [review]

File: _pages/en/strategy-target-solution-delivery-model.md (hunk 1 above)
Comment (remyb2canada) [reply _pages/en/strategy-target-solution-delivery-model.md:824]: nice catch!

File: _pages/en/strategy-target-solution-delivery-model.md (hunk 2 above)
Comment (remyb2canada) [reply _pages/en/strategy-target-solution-delivery-model.md:841]: yeah it's a duplicate. It was part of 2 definitions (IT Solution, and IT Product). Each definition re-used the same diagram.

I'll have to update that section in a next version

Comment (ElminaIusifova) [reply _pages/en/strategy-target-solution-delivery-model.md:841]: @remyb2canada I will duplicate the alt tag as well then. Maybe you can update it after?

**Pull request merged**